liblocation/src/main/java/com/mapbox/android/core/location/LocationEngineResult.java

```diff
@@ -37,9 +38,10 @@ private LocationEngineResult(List<Location> locations) {
    * @since 1.0.0
    */
   public static LocationEngineResult create(Location location) {
```

Comment:
As this is a `public` method, could we add `@Nullable` / `@NonNull` annotations to help Kotlin users?